source/module_io/td_current_io.cpp

```diff
@@ -10,135 +10,104 @@
 #include "module_elecstate/module_dm/cal_dm_psi.h"
 #include "module_elecstate/potentials/H_TDDFT_pw.h"
 #include "module_hamilt_lcao/hamilt_lcaodft/LCAO_domain.h"
+#include "module_hamilt_lcao/module_tddft/td_velocity.h"
```

Comment:
Do you have auto test examples to protect the 'output current' function?

Reply:
No. Where should I add one?